wagtail/admin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -198,12 +198,14 @@ footer {
     @include clearfix();
     overflow: hidden;
     padding-top: 1.4em;
+    padding-bottom: 0.65em;
```

Comment:
What is de idea about changing this padding? Why change any paddings? This one trows the breadcrumb/header ratio of balance. Also the breadcrumb text positioning is off.

![screen shot 2018-09-20 at 15 50 03](https://user-images.githubusercontent.com/1969342/45823047-15929880-bced-11e8-9dd3-7190994836b7.png)